src/index.module.scss

```
@@ -1319,3 +1319,5 @@ div .nc-token-balance:not(:last-child) {
     }
   }
 }
+
+@import 'styles/web3auth';
```

Stylelint (17.11.1)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)